src/Sql/Sql/VulnerabilityAssessment/Model/VulnerabilityAssessmentSettingsModel.cs

```diff
@@ -93,18 +93,12 @@ public void ValidateContent()
                 NotificationEmail = NotificationEmail.Where(email => !string.IsNullOrEmpty(email)).ToArray();
 
                 // Validity checks:
-                // 1. Check that EmailAddresses are in correct format 
+                // Check that EmailAddresses are in correct format 
                 bool areEmailAddressesInCorrectFormat = Util.AreEmailAddressesInCorrectFormat(NotificationEmail);
                 if (!areEmailAddressesInCorrectFormat)
                 {
                     throw new Exception(Properties.Resources.EmailsAreNotValid);
                 }
-
-                // 2. check that EmailAdmins is not False and NotificationEmail is not empty
-                if (!EmailAdmins && (NotificationEmail == null || !NotificationEmail.Any()))
```

Comment:
@talhers what's the reason for removing this check?

Reply:
We want to remove email constraints from VA settings, meaning the user will not have to provide an email address.
(until now the user had to provide at least one email address or select EmailAdmins. Removing this check allows the user to provide neither)